lib/trino-orc/src/main/java/io/trino/orc/reader/ByteColumnReader.java

```diff
@@ -230,6 +254,15 @@ private static int[] convertToIntArray(byte[] bytes)
         return values;
     }
 
+    private static long[] convertToLongArray(byte[] bytes)
```

Comment:
Instead of handling the schema evolution in each column reader - can we handle them at `ColumnAdaptation` or at layer so that we could reuse them for other formats as well.

Reply:
Yes, we can. Let me change.

Reply:
@Praveen2112 Updated. Could you take another look when you have time?